tests/integration_tests/releases.py

```diff
@@ -73,6 +73,16 @@ def __lt__(self, other: "Release"):
             raise ValueError(f"{self.os} cannot be compared to {other.os}!")
         return version.parse(self.version) < version.parse(other.version)
 
+    def __eq__(self, other):
```

Comment:
We already have an `__eq__`. Were you meaning to replace that one with this one? Is there a problem with it?